src/blocks/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: var(--size);
+  height: var(--size);
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 > :nth-child(n+5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
You should add all modificators for all numbers (0, 1, 2, 3, 4, 5) from the previous task